adafruit_vs1053.py

```diff
@@ -91,7 +99,7 @@ class VS1053:
     # This is NOT thread/re-entrant safe (by design, for less memory hit).
     _SCI_SPI_BUFFER = bytearray(4)
 
-    def __init__(self, spi, cs, xdcs, dreq):
+    def __init__(self, spi: SPI, cs: DigitalInOut, xdcs: Pin, dreq: Pin) -> None:
```

Comment:
`cs` is the input to the `DigitalInOut` constructor, so it must be `Pin` from the `microcontroler` package.